src/test/kotlin/com/terraformation/backend/customer/model/PermissionTest.kt

```diff
@@ -595,6 +616,100 @@ internal class PermissionTest : DatabaseTest() {
     permissions.andNothingElse()
   }
 
+  @Test
+  fun `system user can perform most operations`() {
```

Comment:
This is maybe _slightly_ out of scope for the PR but not totally. I wanted to add a test to verify/document that the system user can't delete itself, and I realized there was no test for system user permissions at all.